framework/Objects/Meta_Fields.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+namespace JustCoded\WP\Framework\Objects;
+
+/**
+ * Meta_Fields is the base class for acquiring data from custom fields.
+ */
+
+class Meta_Fields {
+
+	/**
+	 * Plugin used for extending of post custom fields.
+	 *
+	 * Available values:
+	 *      'just-custom-fields'
+	 *      'advanced-custom-fields'
+	 *
+	 * @var string
+	 */
+	public $custom_fields_plugin = 'just-custom-fields';
+
+	/**
+	 * Current $post object
+	 *
+	 * @var \WP_Post
+	 */
+	public $post;
+
+	/**
+	 * Current $term object
+	 *
+	 * @var \WP_Term
+	 */
+	public $term;
+
+	/**
```

Comment:
i think we need generic construct method to define which plugins is active: acf / jcf. we can do this by simple check of get_field() function